test/test_qarray.py

```diff
@@ -448,6 +448,14 @@ def test_qtype():
     assert a == jqt.Qtypes.from_str("ket")
     assert a != b
 
+    c = jqt.basis(3, 0)
+    d = jqt.basis(3, 0).dag()
+
+    assert c.isvec()
```

Comment:
does this actually run? the method you added is called `is_vec`, so I would think this would throw an error

Please run pytest before merging.